lib/money/currency.rb

```diff
@@ -201,6 +201,24 @@ def unregister(curr)
         existed ? true : false
       end
 
+      # Returns if a code currency is ISO.
+      #
+      # @param [Symbol] code currency
+      #
+      # @return [Boolean]
+      #
+      # @example
+      #   Money::Currency.is_iso?(:eur)
+      #
+      def is_iso?(id)
+        currencies = load_currencies_iso
```

Comment:
This will load the whole currencies file every time you call this method, which can be potentially expensive. Consider tapping into an existing pre-loaded table